examples/plot_simulate_somato.py

```diff
@@ -57,35 +64,77 @@
 # estimate the location and amplitude of a single current dipole. At the
 # moment, we do not offer explicit recommendations on which source
 # reconstruction technique is best for HNN. However, we do want our users
-# to note that the dipole currents simulate with HNN are assumed to be normal
-# to the cortical surface. Hence, using the option ``pick_ori='normal'``
-# seems to make most sense.
-
-method = "MNE"
+# to note that the dipole currents simulated with HNN are assumed to be normal
+# to the cortical surface. Hence, using the option ``pick_ori='normal'`` is
+# appropriate.
 snr = 3.
 lambda2 = 1. / snr ** 2
-stc = apply_inverse(evoked, inv, lambda2, method=method, pick_ori="normal",
-                    return_residual=False, verbose=True)
+evoked = epochs.average()
+stc = apply_inverse(evoked, inv, lambda2, method='MNE',
+                    pick_ori="normal", return_residual=False,
+                    verbose=True)
 
-pick_vertex = np.argmax(np.linalg.norm(stc.data, axis=1))
+###############################################################################
+# To extract the primary response in primary somatosensory cortex (S1), we
+# create a label for the postcentral gyrus (S1) in source-space
+hemi = 'rh'
+label_tag = 'G_postcentral'
+label_s1 = mne.read_labels_from_annot(subject, parc='aparc.a2009s', hemi=hemi,
+                                      regexp=label_tag,
+                                      subjects_dir=subjects_dir)[0]
+
+###############################################################################
+# Visualizing the distributed S1 activation in reference to the geometric
+# structure of the cortex (i.e., plotted on a structural MRI) can help us
+# figure out how to orient the dipole. Note that in the HNN framework,
+# positive and negative deflections of a current dipole source correspond to
+# upwards (from deep to superficial) and downwards (from superficial to deep)
+# current flow, respectively. Uncomment the following code to open an
+# interactive 3D render of the brain and its surface activation (requires the
+# ``pyvista`` python library). You should get 2 plots, the first showing the
+# post-central gyrus label from which the dipole time course was extracted and
+# the second showing MNE activation at 0.040 sec that resemble the following
+# images:
+# |mne_label_fig|
+# |mne_activity_fig|
```

Comment:
actually if you put this after the commented lines, it would look most natural because the plot should appear after the code. But I won't insist if you disagree ...